Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py

```diff
@@ -51,8 +51,8 @@
 
 upgradeProperties[2017] = {
     '2017' : {
-        'Geom' : 'DB:Extended',
-        'GT' : 'auto:phase1_2017_realistic',
+        'Geom' : 'Extended2017dev',
+        'GT' : 'auto:phase1_2017_hcaldev',
```

Comment:
I presume this will switch pixel geometry (well, everything) to be read from XML. In which pre-release is this PR planned to be integrated? (pre12, pre13; or will the geometry be included in a GT before the pre-release?)

I'm wondering a possible interference with #15883 and #15805 (or actually with their validation).